web-server/routes/glossary.js

```diff
@@ -0,0 +1,28 @@
+const url = require('url')
+const bodyParser = require('body-parser')
+const queries = require('../../database/queries')
+const commands = require('../../database/commands')
+
+module.exports = app => {
+
```

Comment:
I think we want it for all glossary routes `app.use('/glossary', app.ensureTrailingSlash)`

Reply:
This isn't working. We followed the pattern we found in modules.js. We left it as is for further comments.